sdk/eventhub/event-hubs/src/eventHubProducerClient.ts

```diff
@@ -139,6 +139,16 @@ export class EventHubProducerClient {
     this._producersMap = new Map();
   }
 
+  /**
+   * Creates an instance of `EventDataBatch` to which one can add events until the maximum supported size is reached.
+   * The batch can be passed to the `send()` method of the `EventHubProducer` to be sent to Azure Event Hubs.
```

Comment:
```suggestion
   * The batch can be passed to the `sendBatch()` method of the `EventHubProducer` to be sent to Azure Event Hubs.
```

Reply:
Fixed with https://github.com/Azure/azure-sdk-for-js/pull/5964/commits/92eb64d98eb395114ec3f7f761db85106344c2f8